cleveref-usedon.dtx

```diff
@@ -579,33 +617,114 @@
 %	This |clist| contains all options that can be passed to |\cref| which are currently implemented.
 %    \begin{macrocode}
 \clist_new:N \g_@@_Options_clist
-\clist_set:Nn \g_@@_Options_clist {UsedOn, UsedBy, UsedByAndOn}
+\clist_set:Nn \g_@@_Options_clist { UsedOn, UsedBy, UsedByAndOn }
 %    \end{macrocode}
 %\end{variable}
 %
+%    \begin{macrocode}
+\cs_new:Nn \@@_UsedOnMessage:n { }
+\cs_new:Nn \@@_UsedByMessage:n { }
+\cs_new:Nn \@@_UsedByAndOnMessage:nn { }
+\NewDocumentCommand \UsedOnText { m } { \@@_UsedOnMessage:n { #1 } }
```

Comment:
I shall only define `\...MessageText` since it would become an internal command after introducing `\Set...MessageText`. However, for the `\Set...` version I shall define both the `\Set...MessageText` and `\Set...MessageContent` (to be the same) so that users can choose to use their preferred one ;)

Reply:
I shall postpone the definition of `\...Content` when the internal expl3 version of commands are ready.